snomed/com.b2international.snowowl.snomed.api.rest/src/com/b2international/snowowl/snomed/api/rest/domain/SnomedConceptMini.java

```diff
@@ -1,15 +1,19 @@
 package com.b2international.snowowl.snomed.api.rest.domain;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonUnwrapped;
 
 public class SnomedConceptMini {
-
+	@JsonUnwrapped
```

Comment:
It seems these annotations are no longer needed because we are manually unwrapping the relationship source, type and destination fsn. I'll merge this in and remove those.